### PR TITLE
MULE-16411: Fix issues affecting mule-integration tests on Java 11

### PR DESCRIPTION
**Pull request by elrodro83**
* Support Java 11 in soap-service test support

### CODE REVIEW

File: tests/unit/src/main/java/org/mule/tck/junit4/rule/ExternalProcess.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+  public ExternalProcess(String... processCommand) {
+    this.processCommand = processCommand;
+  }
+
+  @Override
+  protected void before() throws Throwable {
+    super.before();
+
+    LOGGER.trace("Starting esternal process: " + asList(processCommand));
```
Comment (marianogonzalez) [inline tests/unit/src/main/java/org/mule/tck/junit4/rule/ExternalProcess.java]: `esternal`

File: tests/unit/src/main/java/org/mule/tck/junit4/rule/ExternalProcess.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    daemonFuture = daemonExecutor.submit(() -> {
+      try {
+        p = pb.start();
+
+        final BufferedReader reader = new BufferedReader(new InputStreamReader(p.getInputStream()));
+        String line = reader.readLine();
+        while (line != null) {
+
+          LOGGER.warn(line);
+          if (line.contains("org.eclipse.jetty.server.ServerConnector: Started ServerConnector")) {
```
Comment (marianogonzalez) [inline tests/unit/src/main/java/org/mule/tck/junit4/rule/ExternalProcess.java]: polemico

File: tests/unit/src/main/java/org/mule/tck/junit4/rule/ExternalProcess.java (hunk 1 above)
Comment (elrodro83) [reply tests/unit/src/main/java/org/mule/tck/junit4/rule/ExternalProcess.java]: esssternal

File: tests/unit/src/main/java/org/mule/tck/junit4/rule/ExternalProcess.java (hunk 2 above)
Comment (juandesi) [reply tests/unit/src/main/java/org/mule/tck/junit4/rule/ExternalProcess.java]: polémico 🌝

Comment (elrodro83) [reply tests/unit/src/main/java/org/mule/tck/junit4/rule/ExternalProcess.java]: image

**Pull request merged**